### ISSUE TITLE
Some question about implement in Net

### ISSUE THREAD

**Issue by JIElite** [MorvanZhou/pytorch-A3C#1]
Hi, Morvan:

In [here](https://github.com/MorvanZhou/pytorch-A3C/blob/master/discrete_A3C.py#L45), why do we need to write `self.eval()`? what does it do in effect?

In the Pytorch Official document, it says 

> Sets the module in evaluation mode. This has any effect only on modules such as Dropout or BatchNorm.

The first question is: in this implementation, it doesn't exist Dropout layer so as BatchNorm layer so `self.eval()` has no effect?

The same situation appears in [line 52](https://github.com/MorvanZhou/pytorch-A3C/blob/master/discrete_A3C.py#L52): `self.train()`

Another question is about the usage of `self.eval()` and `self.train()`. If there exists Dropout layer or BatchNorm layer in our neural network, what does the above code actually do?

**Comment by MorvanZhou** [comment]
If they haven't change the core code and if I remember it right, when using self.eval(), the net will not store values for backpropagation, meaning fast computation.
